joins.sql

```diff
@@ -0,0 +1,90 @@
+SELECT *
+FROM users;
+
+SELECT *
+FROM posts
+WHERE user_id='100';
+
+SELECT *, first_name, last_name
```

Comment:
When using * with two or more tables, you'll need to namespace the *. Otherwise, it'll return all the columns from both tables.